code/modules/uplink/uplink_items.dm

```diff
@@ -1,1665 +1,1666 @@
-	Making someone clumsy will allow them to use clown firing pins as well as Reverse Revolvers. For a laugh try using this on the HOS to see how many times they shoot themselves in the foot!"
-	cost = 1
-	item = /obj/item/dnainjector/clumsymut
-	restricted_roles = list("Clown")
-
-/datum/uplink_item/role_restricted/mimery
-	name = "Guide to Advanced Mimery Series"
-	desc = "The classical two part series on how to further hone your mime skills. Upon studying the series, the user should be able to make 3x1 invisible walls, and shoot bullets out of their fingers. Obviously only works for Mimes."
-	cost = 12
-	item = /obj/item/storage/box/syndie_kit/mimery
-	restricted_roles = list("Mime")
-
-/datum/uplink_item/role_restricted/ez_clean_bundle
-	name = "EZ Clean Grenade Bundle"
-	desc = "A box with three cleaner grenades using the trademark Waffle Co. formula. Serves as a cleaner and causes acid damage to anyone standing nearby. The acid only affects carbon-based creatures."
-	item = /obj/item/storage/box/syndie_kit/ez_clean
-	cost = 6
-	surplus = 20
-	restricted_roles = list("Janitor")
-
-/datum/uplink_item/role_restricted/kitchen_gun
-	name = "Kitchen Gun (TM)"
-	desc = "A revolutionary .45 caliber cleaning solution! Say goodbye to daily stains and dirty surfaces with Kitchen Gun (TM)! Just five shots from Kitchen Gun (TM), and it'll sparkle like new! Includes two extra ammunition clips!"
-	cost = 10
-	surplus = 40
-	restricted_roles = list("Cook", "Janitor")
-	item = /obj/item/storage/box/syndie_kit/kitchen_gun
-
-/datum/uplink_item/role_restricted/kitchen_gun_ammo
-	name = "Kitchen Gun (TM) .45 Magazine"
-	desc = "An extra eight bullets for an extra eight uses of Kitchen Gun (TM)!"
-	cost = 1
-	restricted_roles = list("Cook", "Janitor")
-	item = /obj/item/ammo_box/magazine/m45/kitchengun
-
-/datum/uplink_item/role_restricted/explosive_hot_potato
-	name = "Exploding Hot Potato"
-	desc = "A potato rigged with explosives. On activation, a special mechanism is activated that prevents it from being dropped. The only way to get rid of it if you are holding it is to attack someone else with it, causing it to latch to that person instead."
-	item = /obj/item/hot_potato/syndicate
-	cost = 4
-	restricted_roles = list("Cook", "Botanist", "Clown", "Mime")
-
-/datum/uplink_item/role_restricted/his_grace
-	name = "His Grace"
-	desc = "An incredibly dangerous weapon recovered from a station overcome by the grey tide. Once activated, He will thirst for blood and must be used to kill to sate that thirst. \
-	His Grace grants gradual regeneration and complete stun immunity to His wielder, but be wary: if He gets too hungry, He will become impossible to drop and eventually kill you if not fed. \
-	However, if left alone for long enough, He will fall back to slumber. \
-	To activate His Grace, simply unlatch Him."
-	item = /obj/item/his_grace
-	cost = 20
-	restricted_roles = list("Chaplain")
-	surplus = 5 //Very low chance to get it in a surp
```

Comment:
Make a One of these and have the old one... IDK why your nerffing it in the first place

Reply:
I'm not sure what you are talking about? The Syndicate first aid kit received major buffs and the cost was adjusted.